frontend/src/metabase/dashboard/actions/data-fetching.js

```diff
@@ -325,12 +95,9 @@ export const fetchDashboardCardData =
         return dashcard.id;
       });
 
-      dispatch({
-        type: FETCH_DASHBOARD_CARD_DATA,
-        payload: {
-          currentTime: performance.now(),
-          loadingIds: loadingIds.concat(newLoadingIds),
-        },
+      dispatch(fetchDashboardCardDataAction, {
+        currentTime: performance.now(),
+        loadingIds: loadingIds.concat(newLoadingIds),
       });
```

Comment:
TIL that `dispatch` supports payloads in the second parameter 🤯

Reply:
I don't think that's actually true but let me check again after

Reply:
I don't think it works, unless we have some middlewares that do weird things: https://github.com/reduxjs/redux/blob/05ba9d169eea062f700e1468e3beef0545ab535b/src/createStore.ts#L280-L281

That's probably why the tests are failing

Reply:
yeah that's definitely part of it. I just forgot about one set of parenthesis and here we are

Reply:
Hm. I also realize I didn't get any errors on my IDE. I think it must be due to the fact that it's javascript but I'm still surprised I didn't get any warnings or errors from the linter